src/jit/importer.cpp

```diff
@@ -4989,6 +4989,25 @@ GenTreePtr Compiler::impImportLdvirtftn(GenTreePtr              thisPtr,
         NO_WAY("Virtual call to a function added via EnC is not supported");
     }
 
+#if COR_JIT_EE_VERSION > 460
```

Comment:
@BruceForstall @jkotas Could you guys please comment on whether this block should move under the `if (opts.IsReadyToRun())` block a few lines below or not?
CoreRT compilations are also R2R compilations today, but it seems like the CORERT_ABI is declared independently from FEATURE_READYTORUN_COMPILER